tests/unit/xoopsLib/Xoops/Core/Cache/AccessTest.php

```diff
@@ -1,8 +1,8 @@
 <?php
-namespace Xoops\Core\Cache;
```

Comment:
The main function of a namespace is to eliminate the possible interference and collision that occurs when everything shares PHP's global space.

In core we also use the namespace to facilitate autoloading, but the main purpose is to organize code to prevent inadvertent issues in the global namespace.

Test classes are not protected from global collisions, so they could benefit from namespaces, too. The autoloading aspects may not be important in this case, but the uniqueness could be very useful. Most tests have a name based on the class under test, i.e. SomeClass is tested with SomeClassTest. If in core, suppose we have classes
...\Mailer\Plugin and
...\Auth\Plugin

We now have to make up an artificial way to keep the test names unique since we can't have two global classes named PluginTest. A consistent namespace policy makes that a non-issue.

There are two main strategies I see.

One is to just use the same namespace as the class under test.

Another is to add a Test element in the namespace. In doctrine/dbal for example, tests start in the namespace Doctrine\Tests\DBAL.

I can see advantages in both strategies, but have been using the first one (same namespace.) But, either one would be workable. A consistent strategy for this would be a worthwhile addition to our tests. In the long term, this give us a more solid and maintainable product.

Let's give this some thought, and come up with standard to follow. Namespaces are useful language feature; they offer the same advantages to the tests that they give to the core. We should use them to our advantage.